airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.json

```diff
@@ -21,6 +21,11 @@
         "type": "string",
         "description": "The value of the access token generated. See the <a href=\"https://docs.airbyte.io/integrations/sources/facebook-marketing\">docs</a> for more information",
         "airbyte_secret": true
+      },
+      "include_deleted": {
```

Comment:
can we add a custom integration test that setting this flag to true causes us to actually read deleted records?